custom_components/dreo/diagnostics.py

```diff
@@ -10,28 +10,25 @@
 from typing import Any
 
 from .pydreo import PyDreo
-from .haimports import * # pylint: disable=W0401,W0614
-from .const import (
-    DOMAIN,
-    DREO_MANAGER,
-    LOGGER
-)
+from .haimports import *  # pylint: disable=W0401,W0614
+from .const import DOMAIN, DREO_MANAGER, LOGGER
 
 KEYS_TO_REDACT = {
-    "sn", 
-    "_sn", 
-    "wifi_ssid", 
+    "sn",
+    "_sn",
+    "wifi_ssid",
     "module_hardware_mac",
     "password",
     "_password",
     "username",
     "_username",
     "token",
-    "_token"
+    "_token",
```

Comment:
Is this necessary?

Reply:
No, of course not necessary, just a good practice. Will revert.